.github/workflows/main.yml

```
@@ -60,6 +60,36 @@ jobs:
         run: python -m pip install -U tox
       - name: Run lint
         run: tox -epep8
+  cover:
+    name: cover
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

I don't think we want to use python 3.6 since it's EoL. Maybe lets use the latest, 3.10 instead.

```suggestion
      - name: Set up Python 3.10
        uses: actions/setup-python@v2
        with:
          python-version: 3.10
```